test/lint/lint-python.py

```
@@ -124,7 +124,7 @@ def main():
         exit(1)
 
     mypy_files = subprocess.check_output(FILES_ARGS).decode("utf-8").splitlines()
-    mypy_args = ['mypy', '--ignore-missing-imports', '--show-error-codes'] + mypy_files
+    mypy_args = ['mypy', '--show-error-codes'] + mypy_files
```

Ruff (0.12.2)

Consider `['mypy', '--show-error-codes', *mypy_files]` instead of concatenation

Replace with `['mypy', '--show-error-codes', *mypy_files]`

(RUF005)